### DISCUSSION TITLE
How to create internal proxy using Istio within EKS cluster to route requests from private hosted zone(private domain) to Services.

### DISCUSSION THREAD

**Discussion by nphariharan** [istio/istio#50913] [Q&A]
How to create internal proxy using Istio within EKS cluster to route requests from private hosted zone(private domain) to Services. I am trying for the below integration in my AWS EKS cluster

AWS Infra Architecture:

public domain ----> NLB ---->    Service1 ---> Internal proxy ---->  private domain / Service 1
                                                                                                                    private domain / Service 2

EKS Cluster version: 1.29

**Comment by abasitt** [comment]
You can follow these steps. e.g.
1- create a different ns. e.g. istio-private-ingress
2- deploy a dedicated ingress in the istio-private-ingress with unique labels e.g. istio: istio-private-ingress. [here ](https://istio.io/latest/docs/setup/additional-setup/gateway/) is the example you can follow.
3- all your private istio gateways and vs should be bound to the labels you created in step2.
